s4e-web/cypress/integration/map/map-products.spec.ts

```diff
@@ -1,47 +1,50 @@
 /// <reference types="Cypress" />
 
 import { Login } from '../../page-objects/auth/auth-login.po';
-import { Map } from '../../page-objects/map/map.po';
-import { MapFavorities } from '../../page-objects/map/map-favoritie-products.po';
+import { MapDateSelect } from '../../page-objects/map/map-date-select.po';
+import { MapProducts } from '../../page-objects/map/map-products.po';
 
-context.skip('Map Products', () => {
-  beforeEach(() => {
-    cy.fixture('users/zkMember.json').as('zkMember');
+before(() => {
+  cy.fixture('users/zkMember.json').as('zkMember');  
+});
+
+describe('Map Products', () => {
+
+  beforeEach(function(){
+    cy.server();
+   
+    cy.visit('/login');
+    Login
+      .loginAs(this.zkMember)
+  });
+
+  it('should load the product', function(){
```

Comment:
Wouldn't be better to name it `'should load the Airmass product`?

Reply:
In other tests of adding to favorites, adding new Wms, turning on layer, or downloading data, it does not provide the names of the products in the name of each test. I think is not needed. in this example too.